prow/prowjobs/GoogleCloudPlatform/gcp-guest/compute-image-tools.yaml

```diff
@@ -34,7 +34,7 @@ presubmits:
         - "-project=compute-image-test-pool-001"
         - "-var:sbom_util_gcs_root=gs://gce-image-sbom-util"
         - "-var:media=gs://gce-image-build-resources/windows/iso/SW_DVD9_Win_Server_STD_CORE_2019_1809.1_64Bit_English_DC_STD_MLF_X22-02970.ISO"
```

Comment:
Can you please do the same to the iso file and the other `.exe` files?

Reply:
done